src/TreeSupportTipGenerator.cpp

```diff
@@ -66,7 +66,7 @@ TreeSupportTipGenerator::TreeSupportTipGenerator(const SliceMeshStorage& mesh, T
 
     if (max_overhang_speed < 2)
     {
-        max_overhang_insert_lag_ = std::numeric_limits<coord_t>::max();
+        max_overhang_insert_lag_ = static_cast<size_t>(std::numeric_limits<coord_t>::max());
```

Comment:
why not directly take the max of size_t instead of typecasting?

```suggestion
        max_overhang_insert_lag_ = std::numeric_limits<size_t>::max();
```